fixture esmda_has_run fails sometimes

```
    @pytest.mark.usefixtures("use_tmpdir, opened_main_window")
    @pytest.fixture(scope="module")
    def esmda_has_run(run_experiment):
        # Runs a default ES-MDA run
>       run_experiment(MultipleDataAssimilation)

/Users/runner/work/ert/ert/tests/unit_tests/gui/conftest.py:137: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

experiment_mode = <class 'ert.run_models.multiple_data_assimilation.MultipleDataAssimilation'>

    def func(experiment_mode):
        qtbot = QtBot(request)
        gui = opened_main_window
        with contextlib.suppress(FileNotFoundError):
            shutil.rmtree("poly_out")
        # Select correct experiment in the simulation panel
        simulation_panel = gui.findChild(SimulationPanel)
        assert isinstance(simulation_panel, SimulationPanel)
        simulation_mode_combo = simulation_panel.findChild(QComboBox)
        assert isinstance(simulation_mode_combo, QComboBox)
        simulation_mode_combo.setCurrentText(experiment_mode.name())
    
        # Click start simulation and agree to the message
        start_simulation = simulation_panel.findChild(QWidget, name="start_simulation")
    
        def handle_dialog():
            message_box = gui.findChild(QMessageBox)
            qtbot.mouseClick(message_box.buttons()[0], Qt.LeftButton)
    
        QTimer.singleShot(500, handle_dialog)
        qtbot.mouseClick(start_simulation, Qt.LeftButton)
    
        # The Run dialog opens, click show details and wait until done appears
        # then click it
        qtbot.waitUntil(lambda: gui.findChild(RunDialog) is not None)
        run_dialog = gui.findChild(RunDialog)
    
        qtbot.mouseClick(run_dialog.show_details_button, Qt.LeftButton)
    
>       qtbot.waitUntil(run_dialog.done_button.isVisible, timeout=100000)
E       pytestqt.exceptions.TimeoutError: waitUntil timed out in 100000 milliseconds
```
https://github.com/equinor/ert/actions/runs/6257566619/job/16990457170

--- a/tests/unit_tests/gui/conftest.py
+++ b/tests/unit_tests/gui/conftest.py
@@ -169,7 +169,7 @@ def run_experiment_fixture(request, opened_main_window):
 
         qtbot.mouseClick(run_dialog.show_details_button, Qt.LeftButton)
 
-        qtbot.waitUntil(run_dialog.done_button.isVisible, timeout=100000)
+        qtbot.waitUntil(run_dialog.done_button.isVisible, timeout=200000)
         qtbot.waitUntil(lambda: run_dialog._tab_widget.currentWidget() is not None)
 
         # Assert that the number of boxes in the detailed view is

